datahub-web-react/src/app/ingest/source/utils.ts

```diff
@@ -83,29 +95,38 @@ export const getExecutionRequestStatusDisplayText = (status: string) => {
 export const getExecutionRequestSummaryText = (status: string) => {
     switch (status) {
         case RUNNING:
-            return 'Ingestion is running';
+            return 'Ingestion is running...';
         case SUCCESS:
-            return 'Ingestion successfully completed';
+            return 'Ingestion succeeded with no errors or suspected missing data.';
+        case FLAGGED:
+            return 'Ingestion succeeded, but some data is suspected missing.';
+        case WARNING:
+            return 'Ingestion completed with minor or intermittent errors.';
```

Comment:
this vs flagged is unclear to me...

do we _really_ have two distinct states? if we cannot crisply explain it yet i'd wait on adding it until we can